pkg/instancegroups/instancegroups.go

```diff
@@ -248,6 +248,20 @@ func (r *RollingUpdateInstanceGroup) validateClusterWithDuration(rollingUpdateDa
 	}
 }
 
+// validateClusterWithDurationLoop validates cluster 3 times to make sure cluster does not flap
+func (r *RollingUpdateInstanceGroup) validateClusterWithDurationLoop(rollingUpdateData *RollingUpdateCluster, duration time.Duration) error {
+	var err error
+	for i := 1; i <= 3; i++ {
```

Comment:
I had been thinking 2 was sufficient, but I don't know how the nodes flap (that was why I asked for the kubectl output). If there's usually just one dip into success before the kube-controller-manager or whatever pods get created, then 2 would work. If it makes multiple dips into success, then more would be needed to reduce the probability of the checks coinciding with the temporary dips.

Reply:
updated to two now